### PR TITLE
Update links and typos.

### PR DESCRIPTION
**Pull request by roncohen**
Updates a few links, improves chunking and structure.

### CODE REVIEW

File: docs/guide/apm-getting-started.asciidoc [hunk 1]
```diff
@@ -97,13 +98,14 @@ output:
     hosts: ElasticsearchAddress:9200
 ----------------------------------
 
-
+[float]
 ==== Secure access to the API
 The HTTP API exposed by APM Server listens on `localhost` and port 8200 by default.
 If you change the listen address from `localhost` to something that is accessible from outside of the machine, we recommend setting up firewall rules to ensure that only your own systems can access the API.
-Alternatively, you can use the _secret token and TLS_ to secure access to APM Server API.
+Alternatively, you can use the https://www.elastic.co/guide/en/apm/server/current/_security.html[secret token and TLS] to secure access to APM Server API.
```
Comment (dedemorton) [inline docs/guide/apm-getting-started.asciidoc]: We should use asciidoc attributes to resolve link paths that point to the docs. I've created an issue to track some cleanup work that's required. Doesn't have to happen right away...but before GA.

File: docs/guide/apm-getting-started.asciidoc [hunk 2]
```diff
@@ -172,11 +178,13 @@ $ pip install elastic-apm
...
 The Python agent supports Django and Flask out of the box.
-See _Getting started with Django_ or _Getting started with Flask_ for more information on how to get started.
+See https://www.elastic.co/downloads/apm/django[Getting started with Django] or https://www.elastic.co/downloads/apm/flask[Getting started with Flask] for more information on how to get started.
 
+[float]
 === Set up Kibana
 
 You can use the `dashboards` that are packaged with APM Server, as mentioned above.
 
 ..Screenshot of Kibana plugin..
```
Comment (dedemorton) [inline docs/guide/apm-getting-started.asciidoc]: Did you mean to add a screen shot here?

**Comment by dedemorton** [comment]
Can we start labeling all doc-only PRs with the docs label?

File: docs/guide/apm-getting-started.asciidoc [hunk 3]
```diff
@@ -1,7 +1,8 @@
 [[apt-getting-started]]
```
Comment (simitt) [inline docs/guide/apm-getting-started.asciidoc]: should this be `apm-getting-started` ?

File: docs/guide/apm-getting-started.asciidoc [hunk 4]
```diff
@@ -162,6 +167,7 @@ var apm = require('elastic-apm').start({
 The Node.js agent supports Express, hapi and Koa out of the box. 
 See the _individual agent documentation_ for more information.
```
Comment (simitt) [inline docs/guide/apm-getting-started.asciidoc]: You should also add the `Getting started with Node.js` link here.

**Comment by roncohen** [comment]
Updated. Still need to use asciidoc attributes to resolve link paths when we have a common file to use.

**Pull request merged**